app/graph/mutations/tag_text_mutations.rb

```diff
@@ -1,12 +1,10 @@
 module TagTextMutations
   create_fields = {
     team_id: '!int',
-    teamwide: 'bool',
```

Comment:
Also, need to add a migration to remove the `teamwide` column from the database (`docker-compose exec api bundle exec rails g mutation RemoveTeamwideFromTagTexts`)

Reply:
Cool thanks! Is this a mutation generator? I'll run this and commit the file generated by it?

Reply:
Yeah, it's a generator... it will generate a new mutation, then you edit the mutation by adding this to the `change` method: `remove_column :tag_texts, :teamwide`

Reply:
And then you commit the migration file and the updated `db/schema.rb` (which is updated automatically after you run `docker-compose exec api bundle exec rake db:migrate`)